### DISCUSSION TITLE
Discord Webhook

### DISCUSSION THREAD

**Discussion by CurlyBorly** [RandomNinjaAtk/docker-lidarr-extended#65] [Q&A] [answered]
Is there a way to get the webhook working for Discord? It looks like they require a `content` field in the payload. https://discord.com/developers/docs/resources/webhook#execute-webhook

This script only provides `event` and `message`

**Comment by RandomUser345** [comment, marked as answer]
Makes perfect sense. I'll get to it and create a PR.

**Reply by RandomNinjaAtk** [reply]
Merged, update should be building

**Reply by CurlyBorly** [reply]
Thanks so much! I'll update and see if it works

**Reply by CurlyBorly** [reply]
Ok so I tried setting my arl a bogus value but I did not get a discord message when it failed, should it have logged this case?

`2022-09-10 14:46:26 :: Extended Audio ::  Presents: lidarr-extended (1.0.246)`
...
`2022-09-10 14:47:40 :: Extended Audio :: DEEZER :: deemix client setup verification...
2022-09-10 14:47:42 :: Extended Audio :: DEEZER :: ERROR :: Download failed
2022-09-10 14:47:42 :: Extended Audio :: DEEZER :: ERROR :: Please review log for errors in client
2022-09-10 14:47:42 :: Extended Audio :: DEEZER :: ERROR :: Try updating your ARL Token to possibly resolve the issue...
2022-09-10 14:47:42 :: Extended Audio :: DEEZER :: ERROR :: Exiting...`

**Reply by CurlyBorly** [reply]
I figured out the issue. I set the key as `webhook` instead of `webHook`, got it working now
